### PR TITLE
Checkout and payment integration

### PR DESCRIPTION
**Pull request by sanu29**
Added Single Product Page
Added Loader and Toast
Added Order API and Order Page

### CODE REVIEW

File: src/context/order-context.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import { createContext, useContext } from "react";
+import { useState } from "react";
+import { useEffect } from "react";
```
Comment (zabihshaik) [inline src/context/order-context.jsx]: Line numbers 2, 3 , 4 can be written in a single line.

File: src/context/wishlist-context.jsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import axios from "axios";
 import { createContext, useContext } from "react";
 import { useEffect, useState } from "react";
```
Comment (zabihshaik) [inline src/context/wishlist-context.jsx]: Similarly, here as well.

**Review by zabihshaik — APPROVED** [review]
Code Structure looks good. Just a few comments.
Better to us Prettier for code formatting.

File: src/component/CheckoutBody.jsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    const [price, setPrice] = useState(cart.reduce((acc, item) => {
+        acc =   acc + (item.qty * item.price);
+         return acc}, 0)+50)
+    const {pincodeDes} = UseAddressContext()
+    const {order, setorder, addOrder} = UseOrderContext();
+    const {addAddress, address , removeAddress} = UseAddressContext();
+    const navigate = useNavigate()
+
+
+    if(cart === [] || cart === "" || cart === undefined|| cart.length <1)
```
Comment (ritu9898) [inline src/component/CheckoutBody.jsx:81]: if you're checking cart === [], then no need to check the length < 1

**Review by ritu9898 — APPROVED** [review]
Everything else looks good.

**Pull request merged**